opentracing-spring-web-itest/mvc/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/mvc/MVCJettyITest.java

```diff
@@ -68,129 +55,4 @@ protected TestRestTemplate getRestTemplate() {
         return testRestTemplate;
     }
 
-    @Test
-    public void testFilterException() throws Exception {
```

Comment:
I want these tests back we spent a lot of time handling corner cases to trace all possible error states.